bin/main.ts

```diff
@@ -26,10 +26,8 @@ const mgtGalaxy = new GalaxyStack(cosmos, "Mgt", {
   cidr: "10.0.0.0/22"
```

Comment:
Can move this to cosmos level:
`const cosmos = new CosmosStack(app, "Demo", {
  tld: "cosmos.com",
  env: mgtEnvConfig,
cidr: "10.0.0.0/22"
});`